project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.0.4
+sbt.version=1.1.0-RC2
```

Comment:
Sneaky, do we really want that here? ;)

Reply:
I think this has a chance to fix the javac NPE issue we've seen. Perhaps as different mini PR though?

Reply:
Oops, that wasn't intentional here :)